custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -47,7 +47,7 @@ def _validate_content_type_header(header):
         return ret_mimetype, ret_charset
 
     @staticmethod
-    def _fetch_data_from_request(file_key, logger=None):
+    def _fetch_data_from_request(file_key, logger=None, optional=False):
```

Comment:
I think `resolve_mimetype_by_filename` is going to fail for sparse.colnames file

Reply:
No, it won't . My mistake.

Reply:
It seems to work for me.